### PR TITLE
fixes being able to grind guns with an autolathe

### PR DESCRIPTION
**Pull request by PKPenguin321**
>buy revolver for 90000 TCs
>accidentally click autolathe when you're printing more ammo
>it turns into half a metal sheet's worth of material

### CODE REVIEW

**Comment by kevinz000** [comment]
Git gud?

**Comment by PKPenguin321** [comment]
get a sizable wiener you faggot

**Comment by octareenroon91** [comment]
Seems awkward and imperfect.

How aboqtgiving objects a 'commodity' flag or var for one-click reclamation in autolathe?

**Comment by PKPenguin321** [comment]
or we could just not let people grind up c20s/revolvers/shotguns/whatever into effectively nothing since that makes practically no sense

generally autolathe reclamation is reserved for things that you can already craft/create with an autolathe easily but projectile guns do not fall under that category

**Comment by TheNightingale** [comment]
If you're holding a revolver near an autolathe, it's probably because you're printing 50,000 speedloaders to murder the entire station; and if that's the case, you deserve to have your revolver turned into half a sheet of metal. Karma.

File: code/modules/projectiles/guns/projectile.dm [hunk 1]
```diff
@@ -4,7 +4,6 @@
 	icon_state = "pistol"
 	origin_tech = "combat=2;materials=2"
 	w_class = 3
-	materials = list(MAT_METAL=1000)
```
Comment (ChangelingRain) [inline code/modules/projectiles/guns/projectile.dm]: and honestly this isn't even a full sheet of metal

**Comment by Ninja137** [comment]
Because basing code decisions on being assmad is a good decision. Back it up there, Errorage.

**Comment by PKPenguin321** [comment]
>If you're holding a revolver near an autolathe, it's probably because you're printing 50,000 speedloaders to murder the entire station; and if that's the case, you deserve to have your revolver turned into half a sheet of metal. Karma.

it's not exclusively revolvers you cuck that was just an example

**Comment by TheNightingale** [comment]
> show me on this doll where the revolver touched you

-points to the chest- The armour... it does nothing...

**Comment by Cheridan** [comment]
I accidentally put the pinpointer into the autolathe as captain once. :+1: to restricting certain items.

**Comment by optimumtact** [comment]
You're the one being bothered here @PKPenguin321

**Comment by PKPenguin321** [comment]
i'm in a salty mood for unrelated reasons

**Comment by Aloraydrel** [comment]
why not just make it give you a simple prompt if you want to destroy the item?

**Comment by GunHog** [comment]
This is a very rare event. A salt-fueled PR that is actually a good idea....

**Comment by octareenroon91** [comment]
It's a good idea. I still disagree with the specific solution.

**Comment by PKPenguin321** [comment]
>A salt-fueled PR that is actually a good idea....

It's not even that. I discovered this bug ages ago when I was new to the game and that PR that lets to reclaim spears and stuff just reminded me to do this.

To everybody who's opposed to this: Instantly turning guns into half a metal sheet is literally a bug. I can't imagine why you would want that.

**Comment by TheNightingale** [comment]
How about if we make it three metal sheets?

**Comment by PKPenguin321** [comment]
I'm going to assume you're joking because I actually laughed

**Comment by ChangelingRain** [comment]
If you say this is bad you are a nerd!

**Pull request merged**